armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -190,9 +202,19 @@ def setAssembly(self, a, setFuel=True):
         setFuel : boolean, optional
             Boolean to determine whether or not fuel blocks should have their target components set
             This is useful when target components within a fuel block need to be determined on-the-fly.
+        Tinput2Thot: bool
+            determines if thermal expansion factors should be calculated from c.inputTemperatureInC
+            to c.temperatureInC (True) or some other reference temperature and c.temepratureInC (False)
```

Comment:
Typo in temperature

Reply:
Addressed https://github.com/terrapower/armi/pull/1342/commits/00808d2e8f513d95fbdd3d48fdad495eaaea3ce0